frontend/src/commands.json

```diff
@@ -41,15 +39,22 @@
     "merge_production": {
         "user": {
             "merge": "upsun merge staging",
-            "resources_set": "upsun resources:set \\\n\t--size redis_persistent:0.1 \\\n\t--disk redis_persistent:512 \\\n\t-e main",
+            "resources_set": "upsun resources:set \\\n\t--size redis_service:0.5 \\\n\t-e main",
             "get_url": "upsun url --primary -e main"
         },
         "test": {
-            "merge": "git checkout $DEFAULT_BRANCH && git merge $STAGING_BRANCH && git push --force upsun $DEFAULT_BRANCH",
-            "resources_set": "upsun resources:set \\\n\t--size redis_persistent:0.1 \\\n\t--disk redis_persistent:512"
+            "merge": "upsun merge $STAGING_BRANCH",
+            "resources_set": "upsun resources:set \\\n\t--size redis_service:0.5 \\\n\t-e $DEFAULT_BRANCH"
+        }
+    },
+    "scale": {
+        "user": {
+            "resources_set": "upsun resources:set \\\n\t--size redis_service:0.1 \\\n\t-e main"
```

Comment:
```suggestion
            "resources_set": "upsun resources:set \\\n\t--size \"*:0.1\" \\\n\t-e main"
```

Reply:
We should want all apps and services to scale down for trial spend help since the 0.5 profile is the default for all